src/webfaf/stats.py

```diff
@@ -85,14 +86,14 @@ def date_filter(query):
         if not release_sum:
             continue
 
-        percentage = int(release_sum * 100.0 / total)
+        percentage = int(release_sum * 100.0 // total)
 
         comps = queries.get_report_count_by_component(
             db, release.opsys.name, release.version, history=history)
 
         comp_data = []
         for comp, count in date_filter(comps).all():
-            comp_percentage = int(count * 100.0 / release_sum)
+            comp_percentage = int(count * 100.0 // release_sum)
```

Comment:
dtto

Reply:
It should stay ```/``` because the first division operator is ```float```.